README.md

```diff
@@ -56,10 +56,9 @@ def test_request_to_mock_server(mock_server):
         assert_that("The mock server recorded the request", 
                     imposter, had_request().with_path("/test").and_method("GET"))
 ```
+Imposter will be killed after `with`.
 
-Needs a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html), most easily defined in [`conftest.py`]
-(https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions):
-
+Needs a [pytest fixture](https://docs.pytest.org/en/latest/fixture.html), most easily it can be defined in [`conftest.py`](https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions):
```

Comment:
I'm not sure this is an improvement. Other than that, we are good to go.

Reply:
reverted